security/passage/Portfile

```diff
@@ -0,0 +1,74 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8::et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        FiloSottile passage 1.7.4a1
```

Comment:
this appears to be a fork, which we typically don't prefer. Additionally, the version seems to imply this is an alpha-version; with very, very few exceptions we only package released versions.

Reply:
`passage` *is* a fork, but it is not "upstream with some unapplied fixes", but rather "a new program that has the same interface, but different internals". `pass` uses GnuPG for encryption; `passage` uses `age` (also by FiloSottile) or `rage` (a Rust implementation of `age`). Within the encryption portion of `passage`, at least, it is a hard fork that cannot be ported back to upstream. If there were a new release of `pass` with non-encryption updates, I suspect that they could be ported forward.

As such, I don't see the version as indicative of alpha quality (it would be `1.7.5a1` if that were the case), but rather an indication that it is UX-compatible with `pass` 1.7.4, but implemented with `age` instead of `gpg`. The initial release was `a0` in December 2021 (`1.7.4a1` was released in December 2022). It's not the versioning that I would have chosen (I would have simply reset to 1.0), but it is an approach that I understand because `pass` has a fairly long history and starting your versioning from that which you have hard forked is not uncommon.